docs/user-guide.md

```diff
@@ -139,3 +142,42 @@ kubectl create -f examples/volume/local-pv.yaml
 ```
 kubectl create -f examples/volume/pod-local.yaml
 ```
+
+#### NFS volume
+* create the NFS server
+```
+kubectl create -f nfs-server.yaml
+```
+* get the IP address for the NFS server and update the pod yaml
+```
+kubectl get services
```

Comment:
can we put some copy-to-run commands here?
for the update, we can say replace the server name of `nfs-volume` in `pod-nfs.yaml` (also add a link here) with the nfs-server address got from upon command.